fastify.js

```diff
@@ -408,9 +419,15 @@ function build (options) {
     }
 
     validateBodyLimitOption(opts.jsonBodyLimit)
-    var jsonBodyLimit = opts.jsonBodyLimit || _fastify._jsonBodyLimit
 
-    _fastify.after((notHandledErr, done) => {
+    const onRoute = fastIterator(_fastify._globalHooks.onRoute, _fastify)
+    onRoute(onRouteIterator, opts, (err) => {
+      if (err) throw err
```

Comment:
I'm 👎  on on throwing.

Reply:
log and `return` ?

Reply:
it needs to go into the errors of the boot sequence, i.e. the `done` callback of `after`.